docs/opcodes_details.md

```diff
@@ -68,7 +68,7 @@ They pop and push, or pop and peek and then modify stack in-place.
 
```

Comment:
please update the section about arithmetic ops like so that "see section xyz for detailed complexity of OPCODEs"

Reply:
done